Gemfile

```diff
@@ -5,8 +5,8 @@ gem 'jira-ruby', :group => :development
 
 group :test do
   gem 'rspec'
-  gem 'beaker', '~>1.17.0'
+  gem 'beaker', '~> 2.2'
   if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-    gem 'sqa-utils'
+    gem 'sqa-utils', '> 0.11'
```

Comment:
Do we need to pin this to > 0.11, or can we just `bundle update` to update the Gemfile.lock?